firefox-ios/Client/Frontend/Home/Homepage Rebuild/Bookmark/BookmarkState.swift

```diff
@@ -0,0 +1,13 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Storage
+
+struct BookmarkState: Equatable, Hashable {
```

Comment:
this may changed depending on conversation on slack regarding what to name these types